crates/punktf-lib/src/profile/source.rs

```diff
@@ -0,0 +1,124 @@
+//! Provides definitions for the `punktf` source directory.
+//!
+//! The source directory is the central repository used to store
+//! [`Profile`s](`crate::profile::Profile`) and [`Dotfile`s](`crate::profile::dotfile::Dotfile`).
+//! `punktf` will only read data from these directories but never write to them.
+//!
+//! The current structure looks something like this:
+//!
+//! ```text
+//! root/
+//! + profiles/
+//!   ...
+//! + dotfiles/
+//!   ...
+//! ```
+
+use color_eyre::eyre::Context as _;
```

Comment:
what does this do? as _?

Reply:
To use traits on types, the trait must be imported with `use`. But as the trait object itself is never explicitly used, it will pollute the current environment.
Meaning that the Type `Context` is already taken in the current scope (no (new) type with that name can be in this scope).

By assigning the import to `_` the trait is brought into scope but the name is not taken/assigned.

This only makes sense for Traits which are called (e.g. std::io::Write => file.write()) but not implemented (e.g. impl Write for X)

Reply:
Makes sense! thanks for the explanation.